interfaces/vrep/DQ_VrepInterface.m

```diff
@@ -488,6 +527,83 @@ function set_joint_target_positions(obj,handles,thetas,opmode)
                 thetas(joint_index) = double(tmp);
             end
         end
+
+        %% Get joint velocities
+        function [joint_velocities,retval]=get_joint_velocities(obj,handles,opmode)
```

Comment:
The documentation must be improved following the DQ Robotic's standards. The function must have a descriptive name, and all parameters and returned values must be duly explained.

The method `get_joint_positions` above has the same issue. Please, use this opportunity to fix it too.

Reply:
I updated the documentation and the signature of the method. Now, the method matches the C++ signature and has better documentation.  Initially, I implemented the signature 
```matlab
[joint_velocities,retval]=get_joint_velocities(obj,handles,opmode)
```
 following the current style of the class.  Now the signature (and the documentation) is as follows

```matlab
 function joint_velocities = get_joint_velocities(obj,jointnames,opmode)
            % This method gets the joint velocities.
            % Usage:
            %      joint_velocities = get_joint_velocities(jointnames, opmode)   
            %          jointnames: The joint names.
            %          (optional) opmode: The operation mode.
            %
            % Example:
            % jointnames={'LBR4p_joint1','LBR4p_joint2','LBR4p_joint3','LBR4p_joint4',...
            %             'LBR4p_joint5','LBR4p_joint6','LBR4p_joint7'};
            % joint_velocities = get_joint_velocities(jointnames);

               .
               .
               .

              
            end
        end 
```

Please let me know if additional modifications are required.